docs/extend/extend-apiml/service-information.md

```diff
@@ -0,0 +1,112 @@
+# Obtaining Information about API Services
+
+## Overview
+
+API clients can obtain obtain information about API services.
+
+This is done for several use cases:
+
+- Display available services to the user based on some criteria (API ID, hostname, or custom metadata)
+- Locate a specific API service based on specific criteria (for example the API ID)
+- Obtain information that allows routing via API gateway such as _baseUrl_ or _basePath_
+- Obtain information about the API services, its APIs, or its service instances
+
+Notes:
+
+- _baseUrl_ or _basePath_ are explained in [Components of URL](api-mediation-components-of-URL.md)
+- The _API ID_ uniquely identifies the API in the API ML. The API ID can be used to locate the same APIs that are provided by different service instances. The API developer defines this ID. It must be a string of up to 64 characters that uses lowercase alphanumeric characters and a dot: `.`. The Zowe APIs start with the `zowe.` prefix. API ML APIs start with the `zowe.apiml.` prefix.
```

Comment:
I am not sure whether details about API ID belongs here. The audience should be the client developer, not the API developer. It causes distraction in my opinion.

Reply:
You are right. I will remove the extra information.